eth-contracts/contracts/service/DelegateManager.sol

```diff
@@ -213,15 +216,16 @@ contract DelegateManager is RegistryContract {
     }
 
     // Finalize undelegation request and withdraw stake
-    function undelegateStake() external returns (uint newTotal) {
+    function undelegateStake() external isInitialized returns (uint newTotal) {
         address delegator = msg.sender;
 
         // Confirm pending delegation request
         require(
```

Comment:
this require statement is repeated, maybe we could move to a modifier. not blocking though